documentation-website/src/components/FaqItem.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {
+  useState,
+} from 'react';
+import {
+  ONE,
+} from '../constants';
+import {
+  Lang,
+} from './lang';
+
+interface FaqItemProps {
+    id: number;
+    isOpenInitially: boolean;
+}
+
+const FaqItem = ({
+  id, isOpenInitially,
+}:FaqItemProps,) => {
+  const [
+    isOpen,
+    setIsOpen,
+  ] = useState(isOpenInitially,);
+
+  const toggleOpen = () => setIsOpen(! isOpen,);
+
+  return (
+    <div key={`faq${ id }`}>
+      <div onClick={toggleOpen}>
+        <div className="accordion-section">
+          <div>
+            <h5>
+              <Lang lnkey={`home.faq.questions.title_${ id + ONE }`} />
```

Comment:
you will need to either suppress this or change the logic. For suppression:

```ts
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-expect-error
````